core/jobs.py

```diff
@@ -967,6 +1274,10 @@ def stop_computation(cls, user_id, unused_test_mode=False):
         """Cancels the currently-running batch job.
 
         No further batch runs will be kicked off.
+
+        Args:
+            user_id: str. The id of the user stopping the job.
+            unused_test_mode: bool. TODO: Why is this here?
```

Comment:
I have no clue what the aim of this was, it isn't ever called with True and the method itself is never overloaded. Should I remove this?